en/7-iterate-improve-frequently.md

```diff
@@ -5,23 +5,6 @@ lang: en
 altLang: fr
 altLangPage: 7-effectuer-iterations-ameliorations-constamment
 ---
-<div class="dpgn-section-intro-standard">
-
-**[TODO: Add/revise introductory text]**
-
-</div>
-
-<div class="dpgn-section-guidelines">
-
-**Guidelines:**
-
-<!-- markdownlint-disable MD032 -->
-- TOC
```

Comment:
This should be left as is as it auto-generates the table of contents (same page links to the guidelines). 

If you want to have output without such links or with a different layout/design/structure, then you can create a custom "view" (see [Digital Playbook views documentation](https://canada-ca.github.io/digital-playbook-guide-numerique/docs/en/views.html) for details).